chromium_src/components/signin/public/base/signin_switches.cc

```diff
@@ -7,6 +7,14 @@
 
 #include "base/feature_override.h"
 
+namespace switches {
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{
+    {kSyncEnableBookmarksInTransportMode, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
do we have plans support this? Presumably at some point the feature will be removed?

Reply:
As I understand, this is for bookmarks stored in Google Account.
So we don't plan to support this. If the feature will be removed - we will have to back-port the implementation to use the old way to sync bookmarks.

Here are some screenshots of how does it look on Android:
1|2|3
--|--|--
![Screenshot_20250313-142316](https://github.com/user-attachments/assets/72039be7-cfaa-4327-af73-5e54688f097b)|![Screenshot_20250313-142410](https://github.com/user-attachments/assets/aad99b9b-875f-47f3-8409-3277acf5f033)|![Screenshot_20250313-142356](https://github.com/user-attachments/assets/76467ab8-7f0f-4569-b495-33cf3321338f)